client/src/components/Sub_ProductDetail/Sub_StyleSelector/StyleEntry.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+import { AiFillCheckCircle } from 'react-icons/ai';
+
+function StyleEntry({ style, currentStyle, setCurrentStyle, startSelect, setStartSelect}) {
+  // console.log('SHOW CURRENT STYLE: ', currentStyle);
+  const img = style.photos[0];
+
+  return (
+    <div
+      className="style-entry-container"
+      style={{ backgroundImage: `url(${img.thumbnail_url ? img.thumbnail_url.substring(img.thumbnail_url.indexOf('http')) : '../../../assets/noProdImg.png'})` }}
```

Comment:
perhaps you could destructure the thumbnail_url out of the img since it's being used 3 times in a row here?

Reply:
great idea!